src/Utils.js

```diff
@@ -10,6 +10,10 @@ import {
   REACT15,
 } from './version';
 
+function equality(a, b) {
```

Comment:
sure, comparing lengths will always work with `===` :-) but a generic function named "equality" should probably be actually generic.